src/components/atoms/TaskCounter.vue

```diff
@@ -0,0 +1,20 @@
+<script setup>
+import { useStore } from "vuex";
+import { computed } from "vue";
+
+const store = useStore();
+const incompleteCounter = computed(() => store.getters.getIncompleteTodosCount);
+</script>
+
+<template>
+  <span class="counter-label">{{ incompleteCounter }} tasks left</span>
```

Comment:
How it will looks if you will have 0 or 1 task?

Reply:
fixed it to have grammatically appropriate labels